CHANGELOG_PENDING.md

```diff
@@ -8,6 +8,9 @@
 
 - [codegen] - Include properties with an underlying type of string on Go provider instances.
 
+- [cli] - Provide a more useful error instead of panicking when codegen fails during import.
```

Comment:
```suggestion
- [cli] - Provide a more helpful error instead of panicking when codegen fails during import.
```

Minor: this can probably be in the "bug fixes" category?

Reply:
Heh, I put it in improvements because I felt like we aren't actually fixing the bug, but rather improving the error reported by the bug. But yeah, could go either way I suppose :)